pithos/application.py

```diff
@@ -85,26 +87,48 @@ def do_command_line(self, command_line):
         # http://stackoverflow.com/questions/1943747/python-logging-before-you-run-logging-basicconfig
         logging.root.handlers = []
 
+        # Show the Pithos log since last reboot and exit
+        if options.contains('last-logs'):
+            try:
+                from systemd import journal
+                from os.path import basename
+            except ImportError:
+                self._print(command_line, 'Systemd Python module not found')
+                return 0
+
+            # We want the version also since the logging plugin misses
+            # logging messages before it's enabled.
+            self._print(command_line, 'Pithos {}'.format(self.version))
+
+            reader = journal.Reader()
+            reader.this_boot()
+            reader.add_match(SYSLOG_IDENTIFIER='io.github.Pithos')
+
+            _PRIORITY_TO_LEVEL = {
+                journal.LOG_DEBUG: 'DEBUG',
+                journal.LOG_INFO: 'INFO',
+                journal.LOG_WARNING: 'WARNING',
+                journal.LOG_ERR: 'ERROR',
+                journal.LOG_CRIT: 'CRTICIAL',
+                journal.LOG_ALERT: 'ALERT',
+            }
+
+            for entry in reader:
```

Comment:
OK makes sense. Not everyone would clue into the fact that it only outputs logs for that boot.(myself included,lol!!!)